spec/form_models/candidate_specification_form_spec.rb

```diff
@@ -3,8 +3,80 @@
   subject { CandidateSpecificationForm.new({}) }
 
   describe 'validations' do
-    it { should validate_presence_of(:education) }
+    # it { should validate_presence_of(:education) }
```

Comment:
commented code? 👀

Reply:
Thanks for spotting that! @dushan-madetech

Reply:
we could have a lot of fun with `rubocop-rspec` :)
https://github.com/rubocop-hq/rubocop-rspec